exporters/otlp/internal/transform/metric.go

```diff
@@ -289,11 +289,74 @@ func Record(r export.Record) (*metricpb.Metric, error) {
 		}
 		return gauge(r, value, time.Time{}, tm)
 
+	case aggregation.ExactKind:
+		e, ok := agg.(aggregation.Points)
+		if !ok {
+			return nil, fmt.Errorf("%w: %T", ErrIncompatibleAgg, agg)
+		}
+		pts, err := e.Points()
+		if err != nil {
+			return nil, err
+		}
+
+		return gaugeArray(r, pts)
+
 	default:
 		return nil, fmt.Errorf("%w: %T", ErrUnimplementedAgg, agg)
 	}
 }
 
+func gaugeArray(record export.Record, points []otel.Number) (*metricpb.Metric, error) {
+	desc := record.Descriptor()
+	m := &metricpb.Metric{
+		Name:        desc.Name(),
+		Description: desc.Description(),
+		Unit:        string(desc.Unit()),
+	}
+
+	start := record.StartTime()
+	end := record.EndTime()
```

Comment:
Probably worth declaring these variables as the return value of `toNanos` if they are both going to be used that way anyways.

```suggestion
	start := toNanos(record.StartTime())
	end := toNanos(record.EndTime())
```

Reply:
I inlined the variable, it makes more sense in the switch. Also aligns with other transform functions.